pkgs/by-name/xd/xdg-desktop-portal-termfilechooser/package.nix

```
@@ -9,16 +9,17 @@
   inih,
   systemd,
   scdoc,
+  nix-update-script,
 }:
-stdenv.mkDerivation {
+stdenv.mkDerivation (finalAttrs: {
   pname = "xdg-desktop-portal-termfilechooser";
-  version = "0.4.0";
+  version = "1.0.5";
 
   src = fetchFromGitHub {
     owner = "hunkyburrito";
     repo = "xdg-desktop-portal-termfilechooser";
-    rev = "c35af27e323a492cbb3b19bdd135657ae523caef";
-    hash = "sha256-9bxhKkk5YFBhR2ylcDzlvt4ltYuF174w00EJK5r3aY0=";
+    rev = "v${finalAttrs.version}";
```

nit, but since you're touching that line anyway...

```suggestion
    tag = "v${finalAttrs.version}";
```